rules/windows/process_creation/proc_creation_win_service_stop.yml

```diff
@@ -29,7 +29,9 @@ detection:
             - '\pwsh.exe'
         CommandLine|contains: 'Stop-Service '
     filter:
-        CommandLine: 'sc  stop KSCWebConsoleMessageQueue' # kaspersky Security Center Web Console double space between sc and stop
+        CommandLine: 
+             - 'sc  stop KSCWebConsoleMessageQueue' # kaspersky Security Center Web Console double space between sc and stop
+             - 'sc  stop LGHUBUpdaterService' # Logitech LGHUB Updater Service
```

Comment:
Need a double space between sc and stop too?

Reply:
Yes, need a double space there for this condition as well

```
	"CommandLine": "sc  stop LGHUBUpdaterService",
```